### DISCUSSION TITLE
Custom Banner/Footer from config file

### DISCUSSION THREAD

**Discussion by Scely** [lowlighter/denolithic#17] [Ideas]
- Display a banner with custom links from config.yml, next to the document title.
- If specified, add a signature at footer element, visible for all slides, supporting images / text.

**Comment by lowlighter** [comment]
I think we should keep using meta-instructions for header menu and use the links of config.yml for footer instead

My reasoning is that header is more something related to current presentation (you could for exemple link to others resources, related exercises, etc.) while the footer may contain more legal or organnizational stuff (like links to a virtual campus, org site, copyright, licenses, etc.) which is more suitable to config.yml since they would be common to whole instance
